contrib/mod_tls.c

```diff
@@ -8881,8 +8881,8 @@ static void tls_end_sess(SSL *ssl, conn_t *conn, int flags) {
 
     /* 'close_notify' not already sent; send it now. */
     pr_trace_msg(trace_channel, 17,
-      "shutting down TLS session, 'close_notify' not already sent; "
-      "sending now");
+      "shutting down %s TLS session, 'close_notify' not already sent; "
+      "sending now", ssl == ctrl_ssl ? "control" : "data");
```

Comment:
Updating the trace logging for `tls_end_sess()` to more clearly indicate which kind of TLS session is being shut down, for future debugging, proved very useful.